src/blockchain/log-processors/order-created.ts

```diff
@@ -26,9 +28,11 @@ export async function processOrderCreatedLog(augur: Augur, log: FormattedEventLo
     const maxPrice = marketsRow.maxPrice!;
     const numTicks = marketsRow.numTicks!;
     const tickSize = numTicksToTickSize(numTicks, minPrice, maxPrice);
+    const numOutcomes = marketsRow.numOutcomes!;
```

Comment:
I think we need to add "numOutcomes" to the query above here as well

Reply:
Did this in my commit

Reply:
guhhhh